src/new/wallet.js

```diff
@@ -1676,13 +1784,13 @@ class HathorWallet extends EventEmitter {
    *                                                                    to be from another wallet
    * @param {string[]|null} [options.data=null] list of data strings using utf8 encoding to add each as a data script output
```

Comment:
Why don't you use the `CreateTokenOptions` you created?

Reply:
Nice catch!